models/spike_detector.cpp

```diff
@@ -45,7 +45,7 @@ nest::spike_detector::spike_detector()
   : Node()
   // record time and gid
   , device_( *this, RecordingDevice::SPIKE_DETECTOR, "gdf", true, true )
-  , user_set_precise_times_( false )
+  , has_user_set_precise_times_( false )
```

Comment:
Dear Hans-Ekkehard, the first time I saw it, I read "user_set" word as a compound adjective. I will revert this change.